docs/pages/machine-id/workload-identity/aws-roles-anywhere.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Configuring Workload ID and AWS Roles Anywhere
+description: Configuring AWS to accept Workload ID certificates as authentication using AWS Roles Anywhere
+---
+
+Teleport's Workload Identity issues flexible short-lived identities in X.509
+certificates. AWS Roles Anywhere allows you to use these certificates to
+authenticate to AWS services.
+
+This can be useful in cases where a machine needs to securely authenticate with
+AWS services without the use of a long-lived credential. This is because the
+machine can authenticate with Teleport without using any shared secrets by
+using one of our delegated join methods.
+
+This implementation differs from Teleport's Application Access for AWS feature
```

Comment:
Since "Application Access for AWS" isn't a term we use in the docs, I would use more general language, e.g., "This implementation differs from using the Teleport Application Service to protect AWS APIs" or something similar.

Reply:
818afe8c07